.github/workflows/release_ci.yml

```diff
@@ -0,0 +1,75 @@
+name: Release - CI
+on:
+  push:
+    paths-ignore:
+      - '**/*.md'
+      - 'mkdocs.yml'
+      - 'src/docs/**/*'
+    branches:
+      - 'dev'
+env:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    name: Build, Test, Deploy
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core on ${{ matrix.os }}
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.404
+    - name: Set build number 
+      if: matrix.os == 'ubuntu-latest'
+      run: echo "BuildNumber=$(( $GITHUB_RUN_NUMBER + 15471 ))" >> $GITHUB_ENV
```

Comment:
@jptissot cloudsmith has two builds of 15472 (latest).

I am guessing this is because you made a new workflow, and the `GITHUB_RUN_NUMBER` started from 0 again for the workflow?

So another merge should increment it?

But will wait for you to check and confirm this, before we merge anything else.

Reply:
Hmmm strange. Maybe that's why it did that. Because I changed the name of the workflow. We should make sure this number gets incremented properly. Or maybe it's because the number starts at 0?

Reply:
Actually I think it's the symbols. Maybe I should disable them? Look at the last ci run. I'm building the symbols along with the packages.

Reply:
Docs say it starts at 1 https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables

But is unique for each workflow name, so because the previous workflow also started at 1, it's going to add up to `1 + 15471` = `15472` on for both of the two runs on the different workflow names.

I suspect it just needs another small test ci pr to be merged to go to the correct version number.

But you wrote it so I let you decide :)

Reply:
The fastest gun in the west @agriffard has decided to test this theory :)

Looks like it' going to `15437` so just the one set of packages duplicated, and they're old now so all g.